extensions/wikia/AdEngine/js/slot/slotTargeting.js

```diff
@@ -106,8 +111,19 @@ define('ext.wikia.adEngine.slot.slotTargeting', [
 		}
 	}
 
+	function getOutstreamData() {
+        var context = adContext.getContext(),
+            getAdserverTargeting = prebid.get().getAdserverTargetingForAdUnitCode,
+			videoTargeting = getAdserverTargeting && getAdserverTargeting(videoSlots[context.targeting.skin]);
```

Comment:
In Oasis `context.targeting.skin` should be always available: https://github.com/Wikia/app/blob/258cf92130916a8d3b6cf5e25177f1ed11c43b13/includes/Skin.php#L25

Mobile-wiki loads our [adsContext.js](https://github.com/Wikia/mobile-wiki/blob/542a1b700f35303e47fdd70ec74e82bb5c8895ca/app/modules/ads.js#L337) which depends on global ads object which in Oasis is set by the [AdContextService](https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L68) which uses the skin property linked above. But in mobile-wiki it's set by [Mercury API](https://github.com/Wikia/app/blob/147eb410f3127bc5112324f7fe7c79ec6471bd32/extensions/wikia/MercuryApi/MercuryApiController.class.php#L224) which means we're safe as well, aren't we? :japanese_ogre: